### ISSUE TITLE
Mood disorder Not Otherwise Specified

### ISSUE THREAD

**Issue by stefaniadv** [obophenotype/human-phenotype-ontology#11144]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@stefaniadv  This is a diagnosis and not a phenotypic observation -- it is outside the scope of HPO.

**Comment by stefaniadv** [comment]
Hi Peter,
I definitely agree with you, but I’ve noticed that other diagnoses exist as
HPO terms.
So my question is about the reason for this discrepancy.
For example, I’ve seen that there is *hyperactivity* (HP:0000752), *reduced
attention regulation* (HP:00520044), but also *ADHD* (HP:0007018).

Regards,
Stefania
